code/modules/projectiles/guns/energy/heavypulserifle.dm

```diff
@@ -12,7 +12,7 @@
 	w_class = ITEM_SIZE_HUGE
 	force = WEAPON_FORCE_PAINFUL
 	projectile_type = /obj/item/projectile/beam/pulse
-	charge_cost = 6000 //1 heavy shot of this with most basic cells. Titanic but with bigger stats basiclly
+	charge_cost = 4000 //1 heavy shot of this with most basic cells. Titanic but with bigger stats basiclly
```

Comment:
I dont get the point of this change eather

Reply:
Thing is worthless as a weapon and expensive to make.

Reply:
I'm - an idiot, took me till now to realize I changed the cost, not the price. I did this 2 times before but fixed it - seems I missed one.